contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -1316,67 +1316,114 @@ pre_transform_target_entry(ResTarget *res, ParseState *pstate,
 			colname_start = pstate->p_sourcetext + res->name_location;
 		}
 
+		/*
+		 * Case 1 : Handle both singlebyte and multibyte aliases when delimited by 
+		 * square bracket(sqb) and double quoutes(dq).
+		 * For instance, queries like : SELECT 1 AS '您对“数据一览“中的车型，颜色，内饰，选装, ';
```

Comment:
The example is neither using square bracket nor using double quotes

Reply:
Resolved.